src/sonic-yang-models/tests/yang_model_tests/yangTest.json

```diff
@@ -642,6 +642,63 @@
 		}
 	},
 
+	"PORT_VALID_AUTONEG_TEST": {
+		"sonic-port:sonic-port": {
+			"sonic-port:PORT": {
+				"PORT_LIST": [{
+					"port_name": "Ethernet8",
```

Comment:
We use ijson to read this file, but still for readability, file should not be very long. Kindly remove description, admin_status, MTU, fec and pfc_asym from config of all new tests. They are already tested with previous tests.